account_partner_aged_statement_webkit/partner_aged_statement_data.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" ?>
+<openerp>
+    <!-- Mail template is done in a NOUPDATE block
+         so users can freely customize/delete them -->
+    <data noupdate="0">
+
+        <!--Email template -->
+        <record id="email_template_aged_statement" model="email.template">
+            <field name="name">Aged Statement Letter</field>
+            <field name="email_from">${(user.email or '')|safe}</field>
+            <field name="subject">${object.company_id.name} - Your aged balance</field>
+            <field name="email_recipients">${object.id or ''|safe}</field>
+            <field name="model_id" ref="account.model_res_partner"/>
+            <field name="auto_delete" eval="True"/>
+            <field name="report_template" ref="partner_aged_statement_report"/>
+            <field name="report_name">
+                ${(object.name or '').replace('/','_')}
+            </field>
+            <field name="body_html"><![CDATA[
+<div style="font-family: 'Lucica Grande', Ubuntu, Arial, Verdana, sans-serif; font-size: 12px; color: rgb(34, 34, 34); background-color: #FFF; ">
+
+    <!--<p>${user.company_id.overdue_msg.replace('\n', '&lt;br/&gt;')} </p>-->
+    % for line in user.company_id.overdue_msg.split('\n'):
+            <p> ${line}<br/> </p>
+    % endfor
+
+    <br/>
+    ${user.name}
+    <br/>
+    <br/>
+
+    <p style="border-left: 1px solid #8e0000; margin-left: 30px;">
+       &nbsp;&nbsp;<strong>REFERENCES</strong><br />
+       &nbsp;&nbsp;Total overdue amount: ${object.credit} ${user.company_id.currency_id.name}<br/>
+    </p>
+
+    <br/>
+    <br/>
+    <div style="width: 375px; margin: 0px; padding: 0px; background-color: #8E0000; border-top-left-radius: 5px 5px; border-top-right-radius: 5px 5px; background-repeat: repeat no-repeat;">
+        <h3 style="margin: 0px; padding: 2px 14px; font-size: 12px; color: #DDD;">
+            <strong style="text-transform:uppercase;">${user.company_id.name}</strong></h3>
+    </div>
+    <div style="width: 347px; margin: 0px; padding: 5px 14px; line-height: 16px; background-color: #F2F2F2;">
+        <span style="color: #222; margin-bottom: 5px; display: block; ">
+        % if user.company_id.street:
+            ${user.company_id.street}<br/>
+        % endif
+        % if user.company_id.street2:
+            ${user.company_id.street2}<br/>
+        % endif
+        % if user.company_id.city or user.company_id.zip:
+            ${user.company_id.zip} ${user.company_id.city}<br/>
+        % endif
+        % if user.company_id.country_id:
+            ${user.company_id.state_id and ('%s, ' % user.company_id.state_id.name) or ''} ${user.company_id.country_id.name or ''}<br/>
+        % endif
+        </span>
+        % if user.company_id.phone:
+            <div style="margin-top: 0px; margin-right: 0px; margin-bottom: 0px; margin-left: 0px; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; ">
+                Phone:&nbsp; ${user.company_id.phone}
```

Comment:
You can also translate `${_("Phone")}` above and `${_("Web")}`. You could also check if the `nbsp` are really necessary.